spec/ios_extract_keys_from_strings_files_spec.rb

```diff
@@ -0,0 +1,229 @@
+require 'spec_helper'
+require 'tmpdir'
+
+describe Fastlane::Actions::IosExtractKeysFromStringsFilesAction do
+  let(:test_data_dir) { File.join(File.dirname(__FILE__), 'test-data', 'translations', 'ios_extract_keys_from_strings_files') }
+
+  describe 'extract the right keys from `Localizable.strings` to all locales' do
+    def assert_output_files_match(expectations_map)
+      expectations_map.each do |output_file, expected_file|
+        expect(File).to exist(output_file), "expect `#{output_file}` to exist"
+        expect(File.read(output_file)).to eq(File.read(File.join(test_data_dir, expected_file))), "expect content of `#{output_file}` to match `#{expected_file}`"
```

Comment:
Nitpick. When those strings are shown, the error already occurred.

```suggestion
        expect(File).to exist(output_file), "expected `#{output_file}` to exist"
        expect(File.read(output_file)).to eq(File.read(File.join(test_data_dir, expected_file))), "expected content of `#{output_file}` to match `#{expected_file}`"
```

RSpec uses the past tense, too. E.g. [here](https://github.com/rspec/rspec-core/blob/ea8554afd1a2b63677c6593059fa8f2476181deb/spec/support/matchers.rb#L13) and [here](https://github.com/rspec/rspec-core/blob/ea8554afd1a2b63677c6593059fa8f2476181deb/spec/support/matchers.rb#L30).